# This patch is needed for ScummVM <= 2.0.0.
# Future versions of ScummVM won't need this patch as this is now
# fixed upstream.

--- ${srcdir}/${_realname}-${pkgver}/backends/platform/sdl/sdl-sys.h
+++ ${srcdir}/${_realname}-${pkgver}/backends/platform/sdl/sdl-sys.h
@@ -36,10 +36,12 @@
 #if !defined(FORBIDDEN_SYMBOL_ALLOW_ALL) && !defined(FORBIDDEN_SYMBOL_EXCEPTION_FILE)
 #undef FILE
 // Solaris has typedef __FILE FILE in several places already
-#if !defined(__sun)
-typedef struct { int FAKE; } FAKE_FILE;
-#define FILE FAKE_FILE
-#endif   // (__sun)
+// The following lines have to be commented out in ScummVM <= 2.0.0 in order to make
+// compilation work in current MSYS2/Mingw-w64 build environments.
+//#if !defined(__sun)
+//typedef struct { int FAKE; } FAKE_FILE;
+//#define FILE FAKE_FILE
+//#endif   // (__sun)
 #endif
 
 #if !defined(FORBIDDEN_SYMBOL_ALLOW_ALL) && !defined(FORBIDDEN_SYMBOL_EXCEPTION_strcasecmp)
